testsuite/preview/expansion/src/test/java/org/wildfly/test/preview/microprofile/telemetry/AbstractOpenTelemetryIntegrationTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.preview.microprofile.telemetry;
```

Comment:
Perhaps don't use package org.wildfly.test.preview.

Using a package with 'preview' in it was probably a mistake in testsuite/preview/basic.

The general goal of stuff in WFP is that at some point it gets promoted to standard WildFly. With the vertx subsystem that could be very quickly. Promoting means moving the tests out of testsuite/preview so I think it's better to use a package structure that fits wherever the tests will end up.